volume/store/store_windows.go

```diff
@@ -0,0 +1,12 @@
+package store
+
+import "strings"
+
+// normaliseVolumeName is a platform specific function to normalise the name
+// of a volume. On Windows, as NTFS is case insensitive, under
+// c:\ProgramData\Docker\Volumes\, the folders John and john would be synonymous.
+// Hence we can't allow the volume "John" and "john" to be created as seperate
+// volumes.
+func normaliseVolumeName(name string) string {
```

Comment:
I'm kind of torn here. Difficult to explain, but hopefully this makes sense. Sure, I agree that conceptually, the local driver makes sense. However, at the store level, without an additional driver interface to know whether the name should be normalised), it would mean that two volumes could still be created as John and john. Do you think it's worth adding another driver interface for that? My concern is that it would break 'out-of-box' volume drivers.